src/index.html

```diff
@@ -3,11 +3,56 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Card</title>
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
     <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <a href="#">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac image"
+          class="card__img"
+        >
+      </a>
+
+      <a href="#" class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </a>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__rate">
+        <div class="card__stars stars">
```

Comment:
```suggestion
        <div class="card__stars stars stars--4">
```
Consider using a structure from the previous task